CHANGES.md

```diff
@@ -1,6 +1,9 @@
 Unreleased
 ----------
 
+- Dune in watch mode no longer builds concurrent rules in serial (#7395
```

Comment:
fixes https://github.com/ocaml/dune/issues/5549 ?

Reply:
We'll claim that we fixed it once we bring back cutoff without the penalty.